resources/authgear/templates/en/web/authflowv2/ldap_login.html

```diff
@@ -0,0 +1,94 @@
+{{ template "authflowv2/__page_frame.html" . }}
```

Comment:
This commit should include ldap_login.go together. Otherwise I have no idea what the context variable is. For example, I have no idea what `$.LDAPUsernameInputError` is.